Gopkg.lock

```diff
@@ -323,6 +323,14 @@
   pruneopts = "NUT"
   revision = "3864e76763d94a6df2f9960b16a20a33da9f9a66"
 
+[[projects]]
+  digest = "1:7a880746686307655232135521a665f5c719013a978cf2d560c050fbb27489f0"
+  name = "github.com/moovweb/rubex"
```

Comment:
could you please check which package introduced this new dependency. It's archived repository...

Reply:
It's enry (https://github.com/src-d/lookout/commit/4c971e88b85593f88336a85b9d3d5ecbfb297706). Looking at the release notes they [already used rubex for a while](https://github.com/src-d/enry/releases/tag/v1.6.0), and recently there has been [some refactor](https://github.com/src-d/enry/pull/156) that must have triggered this new entry in the lock file.

Reply:
thank you!